src/commands/build/mod.rs

```diff
@@ -1,20 +1,64 @@
 pub mod wranglerjs;
 
+use crate::settings::binding::Binding;
+use crate::settings::metadata;
+use crate::settings::project::KvNamespace;
 use crate::settings::project::{Project, ProjectType};
 use crate::{commands, install};
 use std::env;
+use std::fs;
+use std::fs::File;
+use std::io::prelude::*;
 use std::path::PathBuf;
 use std::process::Command;
 
 use crate::terminal::message;
 
+fn write_kv_metadata(maybe_kv_namespaces: &Option<Vec<KvNamespace>>) -> Result<(), failure::Error> {
```

Comment:
is there any important difference between a `None` and a `Vec<KvNamespace>` of length 0?

Reply:
The thing is that the vector is not present until you specify the key in the config. In the webpack backend I have a unwrap_or_else approach, feel free to indicate which one is prefered.